src/components/input-date.vue

```diff
@@ -140,15 +140,21 @@ export default {
       }
     },
     update: function () {
-      const dt = dayjs.utc(this.date, "YYYY-MM-DD", true)
-      if (
-        dt.isValid() &&
-        dt.isAfter(dayjs.utc("1900-01-01", "YYYY-MM-DD", true)) &&
-        dt.isBefore(dayjs.utc())
-      ) {
-        this.$emit("update:modelValue", dt.toDate())
-      } else {
+      if ("" === this.date || "--" === this.date) {
```

Comment:
ici le cas `"" === this.date` ne me semble pas utile car il est impossible, non ?

Reply:
D'après le debugger vue, a l'initialisation de la page, on a undefined-undefined-undefined mais l'erreur affichée est correcte : Ce champ est obligatoire.
Donc je vais supprimer la condition "" === this.date